gremlin-console/src/test/resources/org/apache/tinkerpop/gremlin/console/jsr223/gremlin-server-integration.yaml

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-host: localhost
+host: 0.0.0.0
```

Comment:
Update PR name - it is not about ports, it is about allowing connections from all interfaces. Also need to elaborate on how this resolves the issue.